libbeat/metric/system/process/process.go

```diff
@@ -576,19 +531,22 @@ func (procStats *Stats) getSingleProcess(pid int, newProcs ProcsMap) *Process {
 
 	err = process.getDetails(procStats.isWhitelistedEnvVar)
 	if err != nil {
-		procStats.logger.Debugf("Error getting details for process %s with pid=%d: %v", process.Name, process.Pid, err)
+		procStats.logger.Errorf("Error getting details for process %s with pid=%d: %v", process.Name, process.Pid, err)
```

Comment:
Maybe this was intentionally logged at the debug level? :grimacing: I can imagine this flooding logs in hosts with many short-lived processes.

Reply:
Btw, this reminds me to https://github.com/elastic/beats/issues/9111.

Reply:
Yah, just decided to refactor most of this. Even if this library is going away in 7.16, it's still kind of a mess.